stdpopsim/catalog/EscCol/species.py

```diff
@@ -46,11 +45,24 @@
         )
     )
 
-_genome = stdpopsim.Genome(
-    chromosomes=_chromosomes,
```

Comment:
`_chromosomes` no longer used?

Reply:
in this case it's used for chrom ids

Reply:
Okay - in that case delete `_chromosomes` and do like this:
```
_ploidy_data = {str(i): _species_ploidy for i in genome_data.data["chromosomes"]}
```
Leaving all that unused stuff around makes it look like you set the mutation rate in `_chromosomes`, but that's not used?